### PR TITLE
#786 Review title and layout with review history

### LINKED ISSUE #786: Update Review home page and review page titles to use less space and show Reviewed by you, consolidator by ....
Also update the border of components to have a grey border as in the designs

## Review Home
Screen Shot 2021-06-23 at 11 45 14 AM

## Review Page
Screen Shot 2021-06-23 at 11 44 54 AM

### PR DESCRIPTION
**Pull request by nmadruga**
Fixes #786

### Back-end PR
- ~~[For PR#477-part1](https://github.com/openmsupply/application-manager-server/pull/412)~~ 
- [Back-end#477F-part2](https://github.com/openmsupply/application-manager-server/pull/431)

### Linked PRs
- #792 
- #798 
- #803  - Base branch

### Changes
- Moved `ReviewContainer` and `ReviewHeader` to be only used in `ReviewPage`
- Added `ReviewHomeHeader` to show as in the designs
- Remove link to application from review, it seems to be more confusing than helpful
- **Back** button in `ReviewPage` takes user to `ReviewHome` and **Back** button in `ReviewHome` takes back to list
- Small tweak on borders to show in grey and some borders to show rounded
- Added labels on the top of the review to show the `reviewer` and if it is a Consolidation or a Review.

### Problems
Not really happy with how this border are looking now. If someone could have a look and advise how to fix this please:
Screen Shot 2021-06-23 at 4 08 36 PM
Screen Shot 2021-06-23 at 4 08 28 PM

### CODE REVIEW

**Comment by Chris-Petty** [comment]
Got notified and like, peeked 😅 . Yea borders can be reaaaal fun.

Had a similar problem in mobile in Feb with paper shadows getting cut off badly in mobile. Might be easier to get rid of the container and colour the borders of each of the children directly. It appears the container might be making the children a few pixels smaller than the other elements too:

Screen Shot 2021-06-24 at 11 52 15 AM

I'm not familiar with the designs at all, but the light blue highlight appears to draw users' attention to the area already, so maybe could look cleaner without the hard blue border altogether! That'd be an easy fix 😆 .

Also the alignment of the text elements is a bit off. Text can be a pain, but I'm guessing different flex alignments (web does behave different from react native though so take my rusty input with a massive grain of salt). I've found earlier this year that's better practice making text and buttons flex 0, and using the container handle their alignment and stretch. I think that's what I did in Feb anyway haha.

image

Ok run away byeeeeee!

File: src/containers/Review/ReviewHomeWrapper.tsx [hunk 1]
```diff
@@ -58,9 +67,59 @@ const ReviewHomeWrapper: React.FC<{
...
+        className="simple-label clickable"
+        onClick={() => push(`/applications?type=${templateCode}`)}
+        icon={<Icon name="chevron left" className="dark-grey" />}
+      />
+      <Header
+        as="h2"
+        content={applicationName}
+        subheader={<Header as="h5" content={orgName || strings.TITLE_NO_ORGANISATION} />}
+      />
+    </div>
```
Comment (CarlosNZ) [inline src/containers/Review/ReviewHomeWrapper.tsx:99]: Oops - merge conflict markers still in here.

Comment (nmadruga) [reply src/containers/Review/ReviewHomeWrapper.tsx]: Not sure how did this happen. I had fixed the exact same conflicts yesterday.

**Comment by nmadruga** [comment]
I'm going to merge this one now if that's alright @CarlosNZ

**Comment by CarlosNZ** [comment]
> I'm going to merge this one now if that's alright @CarlosNZ

Sweet -- I assume you've sorted out the conflict markers etc. since I last looked? THanks :)

**Review by CarlosNZ — APPROVED** [review]

**Pull request merged**